_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -100,10 +100,18 @@ This `input` element is not [focusable][] because of the `disabled` attribute.
 This `a` element is not [focusable][] because it moves focus to the `input` element whenever it receives focus.
 
 ```html
+<div role="dialog" aria-label="Sample Modal" aria-modal="true" style="border: solid black 1px; padding: 1rem;">
+    <label>Some input <input id="dialogFirst"></label><br />
+    <button>Close button</button>
+</div>
 <div aria-hidden="true">
-	<a href="/" style="position:absolute; top:-999em" onfocus="document.querySelector('input').focus()">First link</a>
+    <a href="#" id="sentinelAfter" style="position:absolute; top:-999em">Focus sentinel</a>
 </div>
-<input />
+<script>
+    sentinelAfter.addEventListener("focus", () => {
```

Comment:
Oh wow! There's DOM quirkiness you don't see every day.

I know this works, but for ease of understanding, would it be better to replace this with:

```suggestion
    document.getElementById("sentinelAfter").addEventListener("focus", () => {
```